tests/testVue.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+i= basename "$PWD"
+
+if [ $1 == 2 ] || [ $1 == 3 ];
+then
+  ProjectName="test-app-vue$1"
+
+  if [ "$i" != "tests" ]; then
+    cd tests
+  fi
+
+  cleanup() (
+    cd ..
+    rm -rf $ProjectName
+  )
+
+  echo $ProjectName
+
+  vue create $ProjectName --no-git --inlinePreset "{\"useConfigFiles\": true,\"plugins\": {},\"vueVersion\": \"$1\"}" || ERRCODE=$?
+
+  cd $ProjectName
+  yarn add --dev file:../.. || ERRCODE=$?
```

Comment:
Does Vue CLI use yarn by default? Even or people who don't have yarn installed? I would prefer pnpm but don't think vue cli uses it, but maybe npm would be best because everyone who's contributing to this project must have npm installed but is not guaranteed to have yarn installed.

Reply:
Yes @vue/cli uses yarn by default. But pnpm is supported. Changed it in 0ce06e7